pkg/controllers/managementuser/healthsyncer/healthsyncer.go

```diff
@@ -25,6 +27,14 @@ const (
 	syncInterval = 15 * time.Second
 )
 
+var excludedComponentMap = map[string][]string{}
+
+func init() {
+	excludedComponentMap[providers.AKS] = []string{"controller-manager", "scheduler"}
+	excludedComponentMap["tencenttke"] = []string{"controller-manager", "scheduler"}
+	excludedComponentMap["linodelke"] = []string{"controller-manager", "scheduler"}
+}
+
```

Comment:
The init here is not needed, the map can be declared and initialized in the var. See https://github.com/rancher/rancher/blob/master/pkg/controllers/managementuser/rbac/prtb_handler.go#L25 for an example.